src/Command/Config/ExportContentTypeCommand.php

```diff
@@ -65,8 +65,8 @@ protected function interact(InputInterface $input, OutputInterface $output)
         // --content-type argument
         $contentType = $input->getArgument('content-type');
         if (!$contentType) {
-            $entity_manager = $this->getEntityManager();
-            $bundles_entities = $entity_manager->getStorage('node_type')->loadMultiple();
+            $entityTypeManager = $this->getService('entity_type.manager');
```

Comment:
instead of getService method . use $this->entityTypeManager() method of ContainerAwareCommand class.
This change need at all other places.